mixed-juices/package.json

```diff
@@ -18,6 +18,7 @@
     "babel-jest": "^29.4.2",
     "core-js": "~3.27.2",
     "eslint": "^8.34.0",
+    "eslint-config-airbnb": "^19.0.4",
```

Comment:
It looks like you're still missing to commit the changes in the `package-lock.json` file. Please push up those updates as well.

Reply:
Separately, make sure this return statement is included in its own line
https://github.com/JuanVimon3/exercism-javascript/blob/juices/mixed-juices/mixed-juices.js#L61

Reply:
Not a big deal, but it makes the code more readable.

Reply:
![image](https://user-images.githubusercontent.com/111032081/229835096-13ccee16-d2f0-4017-821b-a849a053414c.png)
![image](https://user-images.githubusercontent.com/111032081/229835183-10243515-7d14-49d3-aa1c-c8359ff699bd.png)